package.json

```diff
@@ -79,7 +79,7 @@
         "type-fest": "2.12.2",
         "bcrypto": "5.4.0",
         "react": "18.2.0",
-        "electron": "18.3.5",
+        "electron": "20.3.5",
```

Comment:
why not 21?

Reply:
Not available in stable nix

https://search.nixos.org/packages?channel=22.05&from=0&size=50&sort=relevance&type=packages&query=Electron_21